### DISCUSSION TITLE
Multitouch bug (I believe it is Genymotion's bug)

### DISCUSSION THREAD

**Discussion by UlyssesZh** [Genymobile/feedback#10] [Genymotion Desktop] [answered]
I'm using Genymoiton on Ubuntu 20.04. To see the bug, first have a touchscreen, and then launch a virtual device, and do the following:
1. Enable 'show taps' in developer options;
2. Start swiping with finger 1;
3. While swiping with finger 1, start swiping with finger 2;
4. While swiping with finger 1, release finger 2;
5. Release finger 1.

This is what would be seen:
![Peek 2022-07-13 21-27](https://user-images.githubusercontent.com/26196187/178909689-3ed62fdb-688c-4318-b814-12d9c0028a08.gif)
Unexpected behaviors are seen:
- In step 3, an unexpected tap is shown at the position of finger 2;
- In step 4, an unexpected tap is shown at the position of finger 1.

The extra tap in step 4 is really visible when playing a game that has obvious tapping effects (KALPA for example):
![Peek 2022-07-14 16-55](https://user-images.githubusercontent.com/26196187/178944425-5625e3ac-da5a-4f36-827f-6eedafcc1135.gif)

It is indeed a bug of Genymotion because I find that it is not a bug of the touchscreen device, any drivers, or Xorg.

These extra taps are fatal when playing rhythm games. Here is a 0.5x speed video of me playing Phigros on Genymotion:

https://user-images.githubusercontent.com/26196187/178910348-c558c29e-51ed-4a8d-ab9f-971b9bb6469d.mp4

As can be seen, there is an extra tap at 00:03:16.

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @UlyssesZh,

Thank you very much for your report!

I have forwarded it to our technical team for investigation.
However, I would like to point out that Genymotion Desktop has not been designed nor optimized for playing games (see [Can I play games with Genymotion Desktop?](https://support.genymotion.com/hc/en-us/articles/360003059737)).

Thank you for your understanding.
Best regards,
